### PR TITLE
Fix GPS Staging Android APK builds

### PR DESCRIPTION
**Pull request by JacobJaffe**
The bundle step wasn't occurring for GPS staging!
The generated apk should work now!

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -95,6 +95,7 @@ project.ext.react = [
     entryFile: "index.js",
     bundleInDevDebug: false,
     bundleInStagingDebug: true,
```
Comment (JacobJaffe) [inline android/app/build.gradle]: @efalkner I think we might be able to get rid of `bundleInStagingDebug`, if that's no longer used? double checking it's not used by BT

**Review by johnschoeman — DISMISSED** [review]
Nice!

File: android/app/build.gradle (hunk 1 above)
Comment (mattThousand) [reply android/app/build.gradle]: @JacobJaffe @efalkner presumably `bundleInBteStaging` should be added too? and `bundleInDevDebug` can also be removed as it is no longer a `buildvariantProductflavor` combination (`bundleInGpsDebug` and `bundleInBteDebug` should be added)

File: android/app/build.gradle [hunk 2]
```diff
@@ -95,6 +95,7 @@ project.ext.react = [
...
+    bundleInGpsStaging: true,
     devDisabledInStagingDebug: true,
```
Comment (mattThousand) [inline android/app/build.gradle:99]: `devDisabledInStagingDebug` `devDisabledInGpsStaging`

also add `devDisabledInBteStaging`

File: android/app/build.gradle [hunk 3]
```diff
@@ -95,6 +95,7 @@ project.ext.react = [
     entryFile: "index.js",
     bundleInDevDebug: false,
```
Comment (mattThousand) [inline android/app/build.gradle]:
```
project.ext.react = [
    entryFile: "index.js",
    bundleInGpsDebug: false,
    bundleInBteDebug: false,
    bundleInGpsStaging: true,
    bundleInBteStaging: true,
    devDisabledInGpsStaging: true,
    devDisabledInBteStaging: true,
    enableHermes: false,  // clean and rebuild if changing
]
```

**Review by mattThousand — CHANGES_REQUESTED** [review]

**Review by mattThousand — APPROVED** [review]

**Pull request merged**